UI/data/locale/en-US.ini

```diff
@@ -1229,6 +1229,10 @@ Basic.Settings.Advanced.StreamDelay.MemoryUsage="Estimated Memory Usage: %1 MB"
 Basic.Settings.Advanced.Network="Network"
 Basic.Settings.Advanced.Network.Disabled="The currently selected streaming protocol does not support changing network settings."
 Basic.Settings.Advanced.Network.BindToIP="Bind to IP"
+Basic.Settings.Advanced.Network.IPVersion="IP Version"
```

Comment:
I kind of don't like "IP Version", but I also can't think of some other name that would fit better; i.e. it is correct, but feels slightly weird to have to change "IP Version" if routing for one or the other is bad (i.e. changing "version" as opposed to … something else)

Sorry, I don't really have a constructive idea for this, lol